app/_src/introduction/kuma-requirements.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kuma Requirements
+---
+
+This page expose the different requirements to run {{site.mesh_product_name}}.
+
+## Architecture
+
+{{site.mesh_product_name}} supports machines with `x86_64` architecture and since `1.7.0` it's possible to run {{site.mesh_product_name}} on `arm64`.
+
+## Kubernetes
+
+{{site.mesh_product_name}} is validated against Kubernetes `1.19.x`, `1.20.x`, `1.21.x` and `1.22.x`.
+
+## Envoy
+
+Versions of envoy supported are: `~1.22.0` which means `>=1.22.0` and `<1.23.0`.
```

Comment:
This depends on https://github.com/kumahq/kuma/blob/abf0346477a9e8d4dba0f0c74a1c06fd7d782e83/app/kuma-dp/cmd/run.go#L189 and https://github.com/kumahq/kuma/blob/87233aabaac68ee56787d8e4e360e5614ad02689/tools/releases/version.sh#L11 - also would be good to have this automated.

side note: Do we need to explain SemVer to the users?

Reply:
Added a link to version.sh, I think adding a link to run.go would confuse the reader, let me know if you disagree. Automation is out of scope for this PR, it should be scoped and tackled separately.